### PR TITLE
Moving queries and updating for K8s running locally

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: src/MarketingPageAcceptanceTests.Steps/Steps/AboutSolution/SupplierData_DownloadNHSAssuredIntegrationsAttachment.cs [hunk 1]
```diff
@@ -19,6 +20,7 @@ public SupplierData_DownloadNHSAssuredIntegrationsAttachment(UITest test, Scenar
         public void WhenTheUserChoosesToDownloadTheIntegrationsAttachment()
         {
             var url = _test.pages.PreviewPage.GetNhsAssuredIntegrationsDownloadLinkUrl();
+            Trace.WriteLine($"Roadmap Url: {url}");
```
Comment (stuart-bass-bjss) [inline src/MarketingPageAcceptanceTests.Steps/Steps/AboutSolution/SupplierData_DownloadNHSAssuredIntegrationsAttachment.cs]: trace code intentional?

File: src/MarketingPageAcceptanceTests.Steps/Steps/AuthorityProvidedDataDocument/DownloadAuthorityProvidedDataDocument.cs [hunk 2]
```diff
@@ -19,6 +20,7 @@ public DownloadAuthorityProvidedDataDocument(UITest test, ScenarioContext contex
         public void WhenTheUserChoosesToDownloadTheAuthorityProvidedDataDocument()
         {
             var url = _test.pages.PreviewPage.GetAuthorityProvidedDataDownloadLinkUrl();
+            Trace.WriteLine($"Roadmap Url: {url}");
```
Comment (stuart-bass-bjss) [inline src/MarketingPageAcceptanceTests.Steps/Steps/AuthorityProvidedDataDocument/DownloadAuthorityProvidedDataDocument.cs]: Trace intentional?

File: src/MarketingPageAcceptanceTests.Steps/Steps/ProductRoadmap/SupplierData_DownloadRoadmapAttachment.cs [hunk 3]
```diff
@@ -19,6 +20,7 @@ public SupplierData_DownloadRoadmapAttachment(UITest test, ScenarioContext conte
         public void WhenTheUserChoosesToDownloadTheRoadmapAttachment()
         {
             var url = _test.pages.PreviewPage.GetRoadmapDownloadLinkUrl();
+            Trace.WriteLine($"Roadmap Url: {url}");
```
Comment (stuart-bass-bjss) [inline src/MarketingPageAcceptanceTests.Steps/Steps/ProductRoadmap/SupplierData_DownloadRoadmapAttachment.cs]: Trace

File: src/MarketingPageAcceptanceTests.TestData/Capabilities/EpicDto.cs [hunk 4]
```diff
@@ -13,7 +13,7 @@ public sealed class EpicDto
 
         internal IEnumerable<EpicDto> GetAllByIdPrefix(string connectionString, string id)
         {
-            var query = Queries.GetAllEpicsByCapabilityPrefix;
+            var query = "SELECT * FROM Epic WHERE Id LIKE @ID";
```
Comment (stuart-bass-bjss) [inline src/MarketingPageAcceptanceTests.TestData/Capabilities/EpicDto.cs]: SHould this be "like @ID%" for a prefix?

File: src/MarketingPageAcceptanceTests.Steps/Steps/ProductRoadmap/SupplierData_DownloadRoadmapAttachment.cs (hunk 3 above)
Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests.Steps/Steps/ProductRoadmap/SupplierData_DownloadRoadmapAttachment.cs]: Nope, will be removed shortly

File: src/MarketingPageAcceptanceTests.Steps/Steps/AuthorityProvidedDataDocument/DownloadAuthorityProvidedDataDocument.cs (hunk 2 above)
Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests.Steps/Steps/AuthorityProvidedDataDocument/DownloadAuthorityProvidedDataDocument.cs]: Nope, will be removed shortly

File: src/MarketingPageAcceptanceTests.Steps/Steps/AboutSolution/SupplierData_DownloadNHSAssuredIntegrationsAttachment.cs (hunk 1 above)
Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests.Steps/Steps/AboutSolution/SupplierData_DownloadNHSAssuredIntegrationsAttachment.cs]: Nope, will be removed shortly

File: src/MarketingPageAcceptanceTests.TestData/Capabilities/EpicDto.cs (hunk 4 above)
Comment (PhysicsJon) [reply src/MarketingPageAcceptanceTests.TestData/Capabilities/EpicDto.cs]: Added the prefix '%' into the param as putting it in after the variable name breaks the sql query

**Pull request merged**